src/jobs/listing/listings.module.css

```diff
@@ -75,6 +75,7 @@
 .rendered__markdown__wrapper h3 {
   font-weight: bold;
   font-size: 1.1rem;
+  margin: 2rem 0 -0.8rem 0;
```

Comment:
Hvor kan dette ses? Tror jeg er dårlig til å lete, men finner ingen umiddelbare steder hvor det er brukt h3 i annonsene

Reply:
På de nye annonsene i allfall. for. nyutdannet oppstart neste høst